### ISSUE TITLE
ArangoDBException: Missing property '_id' on simple traversal with gremlin-java

### ISSUE THREAD

**Issue by mpollmeier** [arangodb/blueprints-arangodb-graph#18]
Getting only the edges works fine (`getEdges(Direction.OUT)`), but `getVertices` throws an Exception. Do you have an idea what's wrong here? 
I am running the latest arangodb-2.6.2 on Fedora 22. 

```
import com.tinkerpop.blueprints.Direction;
import com.arangodb.blueprints.*;
Graph graph = ArangoDBGraphFactory.createArangoDBGraph();
graph.getVertex("1").getVertices(Direction.OUT).iterator().next();
```

Disclaimer: I wrote this code in Scala and then manually converted to Java, so excuse me if there are any minor glitches ;)
Here is the full stack trace:

```
com.arangodb.blueprints.client.ArangoDBException: Missing property '_id'
        at com.arangodb.blueprints.client.ArangoDBBaseDocument.checkHasProperty(ArangoDBBaseDocument.java:109)
        at com.arangodb.blueprints.client.ArangoDBBaseDocument.checkStdProperties(ArangoDBBaseDocument.java:114)
        at com.arangodb.blueprints.client.ArangoDBBaseDocument.setProperties(ArangoDBBaseDocument.java:104)
        at com.arangodb.blueprints.client.ArangoDBSimpleVertex.<init>(ArangoDBSimpleVertex.java:35)
        at com.arangodb.blueprints.ArangoDBVertexIterable$1.next(ArangoDBVertexIterable.java:86)
        at com.arangodb.blueprints.ArangoDBVertexIterable$1.next(ArangoDBVertexIterable.java:57)
        at TraversalSpeedTestTP2$.delayedEndpoint$TraversalSpeedTestTP2$1(TraversalSpeedTestTP2.scala:28)
        at TraversalSpeedTestTP2$delayedInit$body.apply(TraversalSpeedTestTP2.scala:14)
        at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
        at scala.App$class.main(App.scala:76)
        at TraversalSpeedTestTP2$.main(TraversalSpeedTestTP2.scala:14)
        at TraversalSpeedTestTP2.main(TraversalSpeedTestTP2.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at sbt.Run.invokeMain(Run.scala:67)
        at sbt.Run.run0(Run.scala:61)
        at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
        at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
        at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
        at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
        at sbt.Logger$$anon$4.apply(Logger.scala:85)
        at sbt.TrapExit$App.run(TrapExit.scala:248)
        at java.lang.Thread.run(Thread.java:745)
null
```

**Comment by a-brandt** [comment]
Hello Michael,

there are some incompatible changes in ArangoDB 2.6. 

I have to build a new version for ArangoDB 2.6 this week.

**Comment by a-brandt** [comment]
I released version 1.0.11.

Please try the new version.

**Comment by mpollmeier** [comment]
that fixed it, thank you!
